conf/uondrs_ada.config

```diff
@@ -0,0 +1,55 @@
+//Profile config names for nf-core/configs
+
+params {
+    // Config Params
+    config_profile_description = 'University of Nottingham DRS - HPC Profile -- provided by nf-core/configs.'
+    config_profile_contact = 'Niraj Shah (@ShahNiraj)'
+    config_profile_url = 'https://forms.office.com/Pages/ResponsePage.aspx?id=7qe9Z4D970GskTWEGCkKHpYhN2mnMuNPm5Y3GQx5rlxUMzJQTDE0OUVSNUtHT0UyU0o2UTRCQzNaQSQlQCN0PWcu'
+
+}
+
+// definition of the slurm executor. Run the pipeline in a node able to submit jobs to a HPC via sbatch
+singularity {
+  enabled = true
+  autoMounts = true
+  // Some processes don't respect the TMP environment variables: https://github.com/nf-core/rnaseq/issues/115
+}
+ 
+process {
+  // Global process config
+  executor = 'slurm'
+  queueSize = 10
+  queue = 'defq'
+  submitRateLimit = '10/2min'
+}
+ 
+process {
+ 
+    // resources for default process execution
+withLabel:process_low {
```

Comment:
This list of process labels isn't exhaustive I think. Do you need to define them? Normally we've been recommending setting max resources in params (https://nf-co.re/docs/usage/getting_started/configuration#max-resources). 

Nextflow has changed for the better since but that haven't yet reached everywhere, so I still think that's a good idea.